content/updates/2021-05/4-Product-completeness-in-the-API.md

```diff
@@ -0,0 +1,37 @@
+---
+pim_announcement_audience:
+- EE
+- CE
+---
+
+# Highlight the most enriched products on your channels
+::: meta-data type="Improvement" features="Productivity" available="late May" in="EE,GE" link-to-doc="https://api.akeneo.com/concepts/products.html#product"
+
+When developing your connectors, you can now ask for the product completeness whenever retrieving products. Thanks to this piece of information, you can easily push forward the most enriched products on your different channels.
```

Comment:
Why "When developing your connectors"? Why not something more general like "When using our API to develop a PIM extension or simply connecting your PIM other applications, you can now filter.." 
Or only "When connecting your PIM to other applications"

Reply:
```suggestion
When connecting your PIM to other applications through the API, you can now ask for the product completeness whenever retrieving products. Thanks to this piece of information, you can easily push forward the most enriched products on your different channels.
```